vassal-app/src/main/java/VASSAL/command/FlareCommand.java

```diff
@@ -59,6 +48,6 @@ public Point getClickPoint() {
   }
 
   public String getMapId() {
-    return id;
+    return flare.getMap().getId();
```

Comment:
We don't need the Id stored in the Command because the Flare already knows it. We don;t actually need the clickpoint either, you could just use the copy in the Flare for that as well.